src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,621 @@
+- It simplifies the separation of data and basic metadata, enabling, for example, the publication of a dataset in a lightweight fashion with access to the data files on request (as implemented by [DataLad](https://www.datalad.org)).
+
+
+<!--
+This block generates a filename templates.
+The inputs for this macro can be found in the directory
+  src/schema/rules/files/raw
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+
+<!-- Link Definitions -->
+### icephys
+{{ MACROS___make_filename_template(
+   "raw",
+   datatypes=["icephys"],
+   suffixes=["icephys", "events", "channels", "electrodes","scans","probes","coordsystem"]
+)
+}}
+### ecephys
+{{ MACROS___make_filename_template(
+   "raw",
+   datatypes=["ecephys"],
+   suffixes=["ecephys", "events", "channels", "electrodes","scans","probes","coordsystem"]
+)
+}}
+
+
+## Participant information
+
+The `participants.tsv` file is located at the root of the data set directory.
+Its presence is RECOMMENDED in order to describe information about the individual subjects (animals) from which the data was recorded.
+It follows the [general BIDS specifications to describe participants](../modality-agnostic-files.md#participants-file).
+
+On top of the existing columns that can be present in this file and that are described in the BIDS specifications (`participant_id`, `species`, `strain`, `strain_rrid`, `sex`, `handedness`, and `age`), we propose to allow adding the following ones:
+
+
+##  Microephys specific files
+
+The following metadata files are REQUIRED for a given microephys session:
+
+1. `[_ses-<session_label>]_probes.tsv`: A REQUIRED file listing information on the device used to acquire the electrophysiology data, such as implant or probe specification, location, material, and others.
+2. `[_ses-<session_label>]_electrodes.tsv`: A REQUIRED file listing information on the points of electrical contact to the tissue, such as impedance, names, relative positions, and others.
+3. `[_ses-<session_label>]_channels.tsv`: A REQUIRED file listing information on the recorded signals, such as preprocessing, filtering, ids, and others.
+
+As with all tsv-based metadata files in BIDS the probes, electrodes and channels tsv files can be accompanied by json sidecar files.
+
+
+## Coordinate System JSON (`*_coordsystem.json`) & Photos of electrode positions (`_photo.jpg`)
+
+This file provides metadata on the coordinate system in which the electrodes are placed.
+This file is **RECOMMENDED**, and the listed required fields below must be included if a `*_coordsystem.json` file is provided.
+
+The coordinate system can be defined using reference pictures, anatomical landmarks, brain images, or a reference atlas.
+For more details, see the [BIDS Coordinate Systems specifications](../appendices/coordinate-systems.md).
+
+
+Fields relating to the microephys probe and electrode positions:
```

Comment:
somewhere here we should adopt (take and adjust, potentially extend) the figure from probeinterfaces:

https://probeinterface.readthedocs.io/en/main/ to visualize relations between electrodes and probes and channels

![image](https://github.com/user-attachments/assets/ae7ede3a-eaea-427f-91b5-7c10d794d14e)